src/autogluon/bench/frameworks/multimodal/exec.py

```diff
@@ -153,13 +159,18 @@ def run(
         evaluate_args["query_data"] = test_data.data[test_data.text_columns[0]].unique().tolist()
         evaluate_args["response_data"] = test_data.data[test_data.image_columns[0]].unique().tolist()
         evaluate_args["cutoffs"] = [1, 5, 10]
+
+    start_time = time.time()
     scores = predictor.evaluate(**evaluate_args)
+    end_time = time.time()
+    predict_duration = int(end_time - start_time)
 
-    timestamp = datetime.now()
     metrics = {
         "problem_type": predictor.problem_type,
+        "utc_time": utc_time,
```

Comment:
Is there time mainly to record when did the benchmark start? If so, we might want to reflect that in the name. Currently, not quite clear what this time is for

Reply:
Yes it is the start time. I was trying to make it more compatible with what `AMLB` has in its `results.csv`.

Reply:
I see. Then we can stick with it